python/django/security/audit/django-rest-framework/missing-throttle-config.yaml

```diff
@@ -8,14 +8,17 @@ rules:
           }
       - pattern: |
```

Comment:
Should this be `pattern-inside` ?

Reply:
It feels counter-intuitive.
If the match needs to be `REST_FRAMEWORK = ...` and `REST_FRAMEWORK` at the same time, the clauses seems either conflicting (can not be both at the same time) or redundant (one is more restrictive than the other).

Reply:
I see why `pattern-inside` would make more sense, as I'm using patterns to step-by-step drill down to a specific part "keyword".
Addressed in 72b013cb.